app/mails/tools.py

```diff
@@ -41,7 +47,9 @@ def get_delay_addresses_from_recipients(recipients):
     """
     delay_addresses = []
     for recipient in recipients:
-        if re.search(r"^(\d+[dmw])", recipient["email"]):
+        if dateparser.parse(
+            recipient["email"].split("@")[0], settings=settings.DATEPARSER_SETTINGS
```

Comment:
Same here - need to verify that the anti-spam keys are removed before passing stuff to the parser

Reply:
yeah, kinda forgot about the keys, will fix that